main/display/shapes.c

```diff
@@ -61,7 +68,9 @@ static uint32_t dispPxL = NULL;
  */
 void initShapes(void)
 {
-    dispPxL = getPxTftFramebuffer();
+#if defined(__XTENSA__)
+    dispPxL = (uint32_t)getPxTftFramebuffer();
```

Comment:
@cnlohr this would do bad things on x64 if you cram a 64 bit pointer into a 32 bit variable. Good thing the emulator doesnt use TURBO macros I guess.